src/drivers/nuttx/lv_nuttx_fbdev.c

```diff
@@ -342,6 +343,13 @@ static int fbdev_init_mem2(lv_nuttx_fb_t * dsc)
         return ret;
     }
 
+    fbmem = mmap(NULL, pinfo.fblen, PROT_READ | PROT_WRITE, MAP_SHARED | MAP_FILE, dsc->fd, 0);
+
+    if(fbmem == MAP_FAILED) {
+        LV_LOG_ERROR("ERROR: ioctl(FBIOGET_PLANEINFO) failed: %d\n", errno);
```

Comment:
```suggestion
        LV_LOG_ERROR("ERROR: mmap failed: %d\n", errno);
```

Please also update the log message at the other `mmap` location too.

Reply:
Done